api/client/webclient/webconfig.go

```diff
@@ -22,17 +22,17 @@ const (
 	// WebConfigAuthProviderOIDCType is OIDC provider type
 	WebConfigAuthProviderOIDCType = "oidc"
 	// WebConfigAuthProviderOIDCURL is OIDC webapi endpoint
-	WebConfigAuthProviderOIDCURL = "/v1/webapi/oidc/login/web?redirect_url=:redirect&connector_id=:providerName"
+	WebConfigAuthProviderOIDCURL = "/v1/webapi/oidc/login/web?connector_id=:providerName&redirect_url=:redirect"
```

Comment:
If `redirect_url` must be the final query param I would put a comment just so that someone doesn't come along and introduce a regression by tacking a new param on the end.

Reply:
done: https://github.com/gravitational/teleport/pull/12100/commits/6159218da9d78948f2de170fb1bdcf9a40159966